client/rpc-api/src/system/mod.rs

```diff
@@ -59,6 +59,17 @@ pub trait SystemApi<Hash, Number> {
 	#[rpc(name = "system_health", returns = "Health")]
 	fn system_health(&self) -> Receiver<Health>;
 
+	/// Returns the base58-encoded PeerId of the node.
+	#[rpc(name = "system_local_peer_id", returns = "String")]
```

Comment:
The pattern is `<namespace>_<camelCaseName>`
```suggestion
	#[rpc(name = "system_localPeerId", returns = "String")]
```